templates/Features/ShareTarget.CaliburnMicro/App_postaction.xaml.cs

```diff
@@ -0,0 +1,33 @@
+﻿//{[{
+using Windows.UI.Xaml.Controls;
+//}]}
+//{**
+// This code block adds the OnShareTargetActivated event handler to the App class of your project.
+//**}
+namespace Param_RootNamespace
+{
+    public sealed partial class App
+    {
+//^^
+//{[{
+
+        protected override void OnShareTargetActivated(ShareTargetActivatedEventArgs args)
+        {
+            // See more about configure share target in UWP
+            // https://docs.microsoft.com/en-us/windows/uwp/app-to-app/receive-data
+            // See also how to share data from you App
+            // https://docs.microsoft.com/en-us/windows/uwp/app-to-app/share-data
+
+            var rootFrame = Window.Current.Content as Frame;
```

Comment:
This block side steps activation / creation of navigation services leaving the app in a mostly uninitialized state. I'd recommend going through the activation handlers.